apps/settings/src/components/PersonalInfo/BirthdaySection.vue

```diff
@@ -57,15 +60,49 @@ export default {
 	},
 
 	computed: {
+		// Canonical BCP-47 locale (e.g., "de-CH") used by the native date input
+		canonicalLocale() {
+			return getCanonicalLocale()
+		},
+		// Intl formatter for localized date display
+		intlDtf() {
+			return new Intl.DateTimeFormat(this.canonicalLocale, {
```

Comment:
No reason to make this computed its not reactive

```suggestion
		// Intl formatter for localized date display
		intlDtf() {
			return new Intl.DateTimeFormat(getCanonicalLocale(), {
```